src/icons/autocad/__tests__/IconAutoCAD-test.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+import IconAutoCAD from '../IconAutoCAD';
+
+describe('icons/autocad/IconAutoCAD', () => {
```

Comment:
Snapshot test helps for icons, see others for example that have snapshot tests

Reply:
will do